ApplySyntax.sublime-settings

```diff
@@ -203,6 +203,14 @@
                 {"file_path": ".*\\.m$"},
                 {"contains": "(^|\n)\\s*%"}
             ]
+        },
+        {
+            "syntax": "TypoScript/TypoScript",
+            "rules": [
+                {"file_path": ".*(\\\\|/)ext_conf_template\\.txt$"},
+                {"file_path": ".*(\\\\|/)ext_typoscript_(setup|constants)\\.txt$"},
+                {"file_path": ".*(\\\\|/)(setup|constants)\\.txt$"}
```

Comment:
@facelessuser : thanks for your feedback! You're right I guess, so I finetuned the regex to match only when the files are within the folders "fileadmin", "typo3" or "typo3conf", typically the places where those files reside.